cosmwasm/contracts/crosschain-registry/src/exports.rs

```diff
@@ -233,43 +276,30 @@ impl<'a> Registries<'a> {
         Ok(hops)
     }
 
+    /// Returns an IBC MsgTransfer that with a packet forward middleware memo that will send the
+    /// coin back to its original chain and then to the receiver in `into_chain`.
+    /// If the receiver `into_chain` is not specified, the chain this function is called on is assumed.
```

Comment:
Can we be more specific on what the assumption is? I believe the assumption is the "into_chain" is the chain in which the denom is currently existing that we are unwrapping from.

Reply:
yeah, that's correct. I'll update the comment